### PR TITLE
Styling article for Chat control.

### PR DESCRIPTION
**Pull request by mariaterzieva12**

### CODE REVIEW

File: controls/chat/styling.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+slug: chat-styling
+---
+
+# .NET MAUI Chat Styling
+
+Any change in the appearance of the `RadChat` components depends on the referenced styles. The default templates contains a `RadBorder` control (used to achieve the rounded edges), an Image control (used for the avatar image only for the single and first messages) and Label for the text message itself.
+
+Here are the provided styles that you can use to customize the different parts of the control:
+
+- `MessageImageStyle`(`TargetType Image`)&mdash;Defines the style referred to the send button image.
```
Comment (didiyordanova) [inline controls/chat/styling.md:16]: `MessageImageStyle` (target type `Image`)&mdash;Defines the style of the outgoing message image.

same applicable for all properties below. also you have to explain that all styles available for the image can be applied to the MessageImageStyle.

File: controls/chat/styling.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
+---
```
Comment (didiyordanova) [inline controls/chat/styling.md:1]: Change the .md file naming to styling.

File: controls/chat/styling.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+
+- `OutgoingBorderStyle` (`TargetType RadBorder`)&mdash;Defines the style of the outgoing message border.
+
+- `DefaultLabelStyle` (`TargetType Label`)&mdash;Defines the default label style.
+
+- `OutgoingLabelStyle` (`TargetType Label`)&mdash;Defines the style of the label of the outgoing message.
+
+## Customize the Chat Entry
+
+You can customize the `ChatEntry` with implicit style. The example below shows how to customize the chat entry in your Chat control.
```
Comment (didiyordanova) [inline controls/chat/styling.md:30]: Add what is the control used for the ChatEntry

File: controls/chat/styling.md [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+
+```XAML
+<Style TargetType="telerik:ChatEntry">
```
Comment (didiyordanova) [inline controls/chat/styling.md:33]: can we use code snippets from sdk app here? if there are replace the xaml with code snippet

File: controls/chat/styling.md [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+                 Size="{OnPlatform Default=24, WinUI=20}" />
+```
+
+```XAML
+ <telerik:RadChat x:Name="chat"
+                  BackgroundColor="{StaticResource ChatBackgroundColor}"
+                  InputAreaBackgroundColor="{StaticResource ChatBackgroundColor}"
+                  SendMessageButtonImage="{StaticResource SendIconSource}"/>
+```
+You cane see th result from the code snippets in the image below:
```
Comment (didiyordanova) [inline controls/chat/styling.md:59]: typos here 
You can see the result

File: controls/chat/styling.md [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+
+![.NET MAUI Chat Styling](images/chat-styling.png)
+
+
```
Comment (didiyordanova) [inline controls/chat/styling.md:63]: Add see also section

File: controls/chat/styling.md [hunk 7]
```diff
@@ -0,0 +1,69 @@
...
+
```
Comment (didiyordanova) [inline controls/chat/styling.md:64]: remove the lines

File: controls/chat/styling.md (hunk 2 above)
Comment (didiyordanova) [reply controls/chat/styling.md:1]: file name is correct, my mistake ;)

File: controls/chat/styling.md [hunk 8]
```diff
@@ -0,0 +1,69 @@
...
+page_title: .NET MAUI Chat Documentation - Styling
+description: Learn how to style the Telerik UI for .NET MAUI Chat control.
+position: 6
+tags: .net maui, telerik .net maui, ui for .net maui, chat, style
+slug: chat-styling
+---
+
+# .NET MAUI Chat Styling
+
+Any change in the appearance of the `RadChat` components depends on the referenced styles. The default templates contains a `RadBorder` control (used to achieve the rounded edges), an Image control (used for the avatar image only for the single and first messages) and Label for the text message itself.
```
Comment (yordan-mitev) [inline controls/chat/styling.md:12]:
```suggestion
Any change in the appearance of the `RadChat` components depends on the referenced styles. The default templates contain a `RadBorder` control (used to achieve the rounded edges), an Image control (used for the avatar of the single and first message), and a Label for the text message itself.
```

File: controls/chat/styling.md [hunk 9]
```diff
@@ -0,0 +1,69 @@
...
+
+Here are the provided styles that you can use to customize the different parts of the control:
```
Comment (yordan-mitev) [inline controls/chat/styling.md:14]:
```suggestion
To customize the different parts of the control, you can use the following built-in styles:
```

File: controls/chat/styling.md (hunk 3 above)
Comment (yordan-mitev) [inline controls/chat/styling.md:30]:
```suggestion
You can customize the `ChatEntry` with an implicit style. The example below shows how to customize the chat entry in your Chat control.
```

File: controls/chat/styling.md [hunk 10]
```diff
@@ -0,0 +1,69 @@
...
+```XAML
+<Style TargetType="telerik:ChatEntry">
+    <Setter Property="BorderBrush" Value="{StaticResource ChatInputAreaStrokeColor}" />
+    <Setter Property="FocusedBorderBrush" Value="{StaticResource ChatInputAreaStrokeColor}" />
+    <Setter Property="BorderThickness" Value="1" />
+    <Setter Property="CornerRadius" Value="8" />
+</Style>
+```
+
+## Customize the send button
```
Comment (yordan-mitev) [inline controls/chat/styling.md:41]:
```suggestion
## Customize the Send Button
```

File: controls/chat/styling.md [hunk 11]
```diff
@@ -0,0 +1,69 @@
...
+
+You can customize the send button as well. The example below shows how to customize the send button in your Chat control.
```
Comment (yordan-mitev) [inline controls/chat/styling.md:43]:
```suggestion
The example below shows how to customize the Send button in your Chat control.
```

**Pull request merged**